src/utils.js

```diff
@@ -268,3 +268,22 @@ export const getFileSizeToClosestByte = (fileSize) => {
   const fileSizeFixedDecimal = Number.parseFloat(size).toFixed(2);
   return `${fileSizeFixedDecimal} ${units[divides]}`;
 };
+
+/**
+ * Create a correct inner path depend on config PUBLIC_PATH.
+ * @param {string} checkPath - the internal route path that is validated
+ * @returns {string} - the correct internal route path
+ */
+export const createCorrectInternalRoute = (checkPath) => {
```

Comment:
[Non-blocking]  Thank you for handling `PUBLIC_PATH` and the trailing slash! 👍🏼 

This came up a lot after the react-router-v6 upgrade, during Quince testing.  It should be standardized in frontend-platform, eventually.